openfisca_us/tests/policy/baseline/demographic/person/ccdf_eligibility.yaml

```diff
@@ -0,0 +1,54 @@
+- name: Asset eligible, age eligible, not enrolled, initial income eligible
+  period: 2022
+  input:
+    is_ccdf_asset_eligible: True
+    is_ccdf_age_eligible: True
+    is_enrolled_in_ccdf: False
+    is_ccdf_initial_income_eligible: True
+  output:
+    is_ccdf_eligible: True
+- name: Asset eligible, age eligible, not enrolled, initial income not eligible
+  period: 2022
+  input:
+    is_ccdf_asset_eligible: True
+    is_ccdf_age_eligible: True
+    is_enrolled_in_ccdf: False
+    is_ccdf_initial_income_eligible: False
+  output:
+    is_ccdf_eligible: False
+- name: Asset eligible, age eligible, enrolled, continuous income eligible
+  period: 2022
+  input:
+    is_ccdf_asset_eligible: True
+    is_ccdf_age_eligible: True
+    is_enrolled_in_ccdf: True
+    is_ccdf_continuous_income_eligible: True
+  output:
+    is_ccdf_eligible: True
+- name: Asset eligible, age eligible, enrolled, continuous income not eligible
+  period: 2022
+  input:
+    is_ccdf_asset_eligible: True
+    is_ccdf_age_eligible: True
+    is_enrolled_in_ccdf: True
+    is_ccdf_continuous_income_eligible: False
+  output:
+    is_ccdf_eligible: False
+- name: Asset not eligible, age eligible, not enrolled, initial income eligible
+  period: 2022
+  input:
+    is_ccdf_asset_eligible: False
+    is_ccdf_age_eligible: True
+    is_enrolled_in_ccdf: False
+    is_ccdf_initial_income_eligible: True
+  output:
+    is_ccdf_eligible: False
+- name: Asset eligible, age not eligible, not enrolled, initial income eligible
```

Comment:
Could you add one more test for not-enrolled and only continuous eligible? This basically tests the false default for initial eligibility.

Reply:
Great catch. I was thinking just New York. Other states can have different criteria between initial and continuous.